impl/src/main/java/org/jboss/cdi/tck/tests/event/observer/priority/contextLifecycleEvent/ApplicationContextLifecycleEventObserverOrderingTest.java

```diff
@@ -46,9 +40,9 @@ public static WebArchive createTestArchive() {
     @Test
     @SpecAssertion(section = OBSERVER_ORDERING, id = "b")
     public void testContextLifecycleEventOrdering() {
-        // assumes the ActionSequence was empty before test execution
-        String[] expectedArray = new String[] { A, B, C, D };
-        ActionSequence.assertSequenceDataEquals(Arrays.asList(expectedArray));
+        // ActionSequence was originally used, but we cannot reliably reset it without extensions
```

Comment:
Can't use `@Before` or something like that?

Reply:
I thought of that but you cannot do it because of how Arq and Weld interoperate. Basically, Weld container is up and running (and hence the events for app context have been fired) long before you get into `@Before` method. But I ended up rewriting the test and that seems to work fine.

Reply:
Aaaa good point, this is for the `@Initialized(ApplicationScoped.class)` events. Yea agree.